projects/dsp-ui/src/assets/style/_viewer.scss

```diff
@@ -6,6 +6,10 @@
   background-color: $black-12-opacity;
 }
 
+.selected-resource {
+  background-color: lightgray;
```

Comment:
Here you could use one of the color variables from config.scss similar to line 6. Use also `$black-12-opacity` or `$black-14-opacity` or `$black-20-opacity`. This will help to keep the design and style almost consistent.

Reply:
Done